src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -334,20 +325,31 @@ public static GraphBuilder forDirectory(CommandLineParameters params, File dir)
         if (builderParams.htmlAnnotations) {
             graphBuilder.addModule(new AnnotationsToHTML(params.build, builderParams.maxHtmlAnnotationsPerFile));
         }
-        graphBuilder.serializeGraph = ( ! params.inMemory ) || params.preFlight;
+        graphBuilder.serializeGraph = (!params.inMemory) || params.preFlight;
 
-        graphBuilder.addModule(new VehicleSharingBuilderModule());
+        tryAddVehicleSharingBuilderModule(graphBuilder);
 
         return graphBuilder;
     }
 
+    private static void tryAddVehicleSharingBuilderModule(GraphBuilder graphBuilder) {
+        if (System.getProperties().containsKey("sharedVehiclesApi")) {
+            graphBuilder.addModule(new VehicleSharingBuilderModule(System.getProperty("sharedVehiclesApi")));
+        } else {
+            graphBuilder.addModule(VehicleSharingBuilderModule.withoutParkingZones());
+            LOG.warn("Building graph without rentable vehicles parking zones. If you want to add parking zones, " +
+                    "please provide program parameter `--sharedVehiclesApi <URL>`");
```

Comment:
Jeśli nie podasz tego parametru to nie ma skąd brać pojazdów w ogóle. Po co dodajesz `VehicleSharingBuilderModule.withoutParkingZones()`?

Reply:
To jest parametr do budowania grafu. Jeśli nie podasz tego parametru, to zbudujesz graf z możliwością oddawania pojazdów w dowolnym miejscu, ale bez stref parkowania - tak się dzieje na przykład w testach integracyjnych.

Reply:
To nie jest domyślne flow, dlatego zrobiłem statyczny konstruktor, żeby każdy zwrócił uwagę na ten przypadek